### PR TITLE
Test for registering new user 7022024

### PR DESCRIPTION
**Pull request by atha123**
hello Admas,
Please review this test for new user registration.

Thanks,
Latha

### CODE REVIEW

File: ssqatest/tests/frontend/latha_frontend_tests/test_50percent_coupon.py [hunk 1]
```diff
@@ -5,14 +5,8 @@
...
-# from demostore_automation.src.configs.MainConfigs import MainConfigs
-# from demostore_automation.src.pages.CartPage import CartPage
-# from demostore_automation.src.pages.Header import Header
-# import time
 import math
-# import pdb
-# pdb.set_trace()
+
 
 pytestmark=[pytest.mark.feregression, pytest.mark.fesmoke, pytest.mark.end_to_end]
```
Comment (admaskinfu) [inline ssqatest/tests/frontend/latha_frontend_tests/test_50percent_coupon.py:11]: This is upto you but I dont think I would consider these tests 'end_to_end'. If it was me I would remove the 'end_to_end' marker.

File: ssqatest/tests/frontend/latha_frontend_tests/test_login_negative.py [hunk 2]
```diff
@@ -17,18 +17,17 @@ def test_invalid_user_login(self):
...
         account.click_login_button()
-        # import pdb; pdb.set_trace()
 
         #Validate the actual message is same as the expected error message.
         expected_msg="Error: The username invalid is not registered on this site. If you are unsure of your username, try your email address instead."
         account.wait_until_error_is_displayed(expected_msg)
 
-
+        # TODO: Checkout what this is
         # account.get_login_form_heading()
```
Comment (admaskinfu) [inline ssqatest/tests/frontend/latha_frontend_tests/test_login_negative.py:26]: just delete this block. It does not look like it is related to the test. And the test looks completed so you can just delete this.

Comment (admaskinfu) [reply ssqatest/tests/frontend/latha_frontend_tests/test_login_negative.py:26]: I think it is just another way of validating the same thing.

File: ssqatest/tests/frontend/latha_frontend_tests/test_register_new_user.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import pytest
```
Comment (admaskinfu) [inline ssqatest/tests/frontend/latha_frontend_tests/test_register_new_user.py:1]: Few comments on this one:
* Why are you automation the same test again? The test already exists (I think it existed on the base): https://github.com/atha123/demostore_frontend_backend_tests/blob/main/ssqatest/tests/frontend/my_account/test_register_new_user.py

* Why are you creating tests in this folder 'latha_frontend_tests'? why did you need a new folder. Why not just do it in 'frontend' folder. Remember this whole framework is yours so when recuriter/manger looks at it they're is not reason to show 'latha' vs 'other'. Eventually there will be 120 tests and 100 of them are what you wrote so technically this is all you. I do not agree in creating a different folder. But not a  blocker. Just my opinion.

**Review by admaskinfu — APPROVED** [review]
Approved but with minor comments.

File: ssqatest/tests/frontend/latha_frontend_tests/test_50percent_coupon.py (hunk 1 above)
Comment (atha123) [reply ssqatest/tests/frontend/latha_frontend_tests/test_50percent_coupon.py:11]: Yes you are correct. I removed the end_to_end mark. Will commit the test.

File: ssqatest/tests/frontend/latha_frontend_tests/test_login_negative.py (hunk 2 above)
Comment (atha123) [reply ssqatest/tests/frontend/latha_frontend_tests/test_login_negative.py:26]: I was wanting to try the other methods too. But you are right, this does not need to be in the commit. I have removed it. Will commit the test.

**Pull request merged**